addons/osfstorage/models.py

```diff
@@ -160,16 +160,29 @@ def delete(self, user=None, parent=None, **kwargs):
         self._materialized_path = self.materialized_path
         return super(OsfStorageFileNode, self).delete(user=user, parent=parent) if self._check_delete_allowed() else None
 
+    def update_folder_regions(self, destination_parent):
```

Comment:
Does this method only make sense for folders? Should it be moved to `OsfStorageFolder`?